homeassistant/components/lutron_caseta/device_trigger.py

```diff
@@ -280,6 +280,73 @@
     }
 )
 
+
+SUNNATA_KEYPAD_2_BUTTON_BUTTON_TYPES_TO_LIP = {
+    "button_1": 1,
+    "button_2": 2,
+}
+SUNNATA_KEYPAD_2_BUTTON_BUTTON_TYPES_TO_LEAP = {
+    "button_1": 1,
+    "button_2": 2,
+}
+LEAP_TO_SUNNATA_KEYPAD_2_BUTTON_BUTTON_TYPES = {
+    v: k for k, v in SUNNATA_KEYPAD_2_BUTTON_BUTTON_TYPES_TO_LEAP.items()
+}
+SUNNATA_KEYPAD_2_BUTTON_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(SUNNATA_KEYPAD_2_BUTTON_BUTTON_TYPES_TO_LIP),
+    }
+)
+
+SUNNATA_KEYPAD_3_BUTTON_RAISE_LOWER_BUTTON_TYPES_TO_LIP = {
+    "button_1": 1,
+    "button_2": 2,
+    "button_3": 3,
+    "raise": 19,
+    "lower": 18,
+}
```

Comment:
How did you test the LIP buttons?

Reply:
If you made them the same as the LEAP buttons you could use https://github.com/bdraco/aiolip to connect to the telnet port and hit each button to find the LIP button number

Reply:
I didn't test the LIP buttons, I just assumed the integration will require them for compatibility.  As far as I know, RA3 isn't exposed via LIP, LEAP only, so they will never exist really.

Reply:
There is no telnet for RA3.  Only LEAP

Reply:
> There is no telnet for RA3. Only LEAP

Hmm.. Need to think about how we should handle this case.

Reply:
Unfortunately, Lutron deprecated the Telnet interface for the RA3 line of products.  So I'm pretty sure that means the LIP button id's will never be exposed. 

I figured this would be the easiest way to maintain compatibility with the existing code.  I've just set the LIP id's to the same as the LEAP id's for all RA3 Keypads.

Please let me know if I can provide any more information to assist.

Reply:
`SunnataKeypad2Button` or `SunnataKeypad_2Button`

Which one is correct?

Reply:
SunnataKeypad_2Button

Reply:
I pushed a fix since there were both in the PR

Reply:
All 3 in the DEVICE_TYPE_SCHEMA_MAP should have the underscore